### ISSUE TITLE
About unused flags

### ISSUE THREAD

**Issue by eyes-and-brain** [project-topaz/topaz#1404]
**_I have:_**

- [x] searched existing issues (http://project-topaz.com/issues/) to see if the issue has already been opened
- [x] checked the commit log to see if the issue has been resolved since my server was last updated

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

The following flag "WWoodsRTenText" is not used anywhere.
In other words, once the flag is set, it remains as a meaningless flag.

https://github.com/project-topaz/topaz/blob/61a2558614fda01fd57767c37f85334999f819c9/scripts/zones/Windurst_Woods/npcs/Rakoh_Buuma.lua#L66-L68

This time, I happened to find such a meaningless flag.
Is it possible that there are many others?

**Comment by EpicTaru** [comment]
Just my thoughts, but it may be related to this line for the Windy Rank 10 final mission (from FFXIclopedia, https://ffxiclopedia.fandom.com/wiki/Moon_Reading):

`Optional: Most NPCs around Heavens Tower will have additional dialogue after the final battle (Kupipi has a different line after the cutscenes as well), along with NPCs around the Windurst Gate Guards and others involved in the storyline, as usual.`

**Comment by MarianArlt** [comment]
Yeah what EpicTaru said. If it is not used anywhere else, it is probably like it was with San d'Oria Rank 10.
Somebody thought about the Epilogue dialogues and cutscene after finishing Rank 10 setting that charVar, but never actually added anything of it. There's supposed to be a final epilogue cutscene after any nations Rank 10. The Windurst one is upon zoning into Windurst Walls from what the CS looks like.

**Comment by MarianArlt** [comment]
Well it is definitely included with "Moon Reading", the Rank 10 Windy mission.

It could be anything at any point though. I don't see that NPC even mentioned in the guides. Could be an optional dialogue or what EpicTaru said. It's also not clear from the name of the variable itself.

It's true that every nation has an epilogue cutscene (see my San d'Oria PR, in the case of Windy it's in Walls I think), but this doesn't seem to be for that to be honest, even says WWoods. The Rank 10 mission finishes by clicking Door: Vestal Chamber in Heaven's Tower, so any epilogue flag should be set there, not at this NPC.
